app/resources/add_note_base.html.template

```diff
@@ -12,6 +12,21 @@
 
 {% load i18n %}
 
+{% if env.enable_javascript %}
+   <script type="text/javascript">
+   jQuery(document).ready(function(){
+     jQuery("div.status-expand").hide();
```

Comment:
Optional: It would be better to put jQuery("div.status-expand") into a variable because it is used many times.